#! /bin/sh /usr/share/dpatch/dpatch-run
## 0004_missing_man_pages.dpatch by  <fabbe@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add man pages for testsaslauthd, sasldbconverter2 and sasltestsuite.

@DPATCH@
diff -urNad trunk~/saslauthd/testsaslauthd.8 trunk/saslauthd/testsaslauthd.8
--- trunk~/saslauthd/testsaslauthd.8	1970-01-01 02:00:00.000000000 +0200
+++ trunk/saslauthd/testsaslauthd.8	2006-07-27 16:20:07.000000000 +0300
@@ -0,0 +1,16 @@
+.\" testsaslauthd - SASL2 test tool
+.\" Dima Barsky 23/12/2002
+.\"
+
+.\"
+.TH TESTSASLAUTHD 8 "December 23, 2002" "CMU SASL"
+.SH NAME
+testsaslauthd \- a test tool for saslauthd
+.SH SYNOPSIS
+.B testsaslauthd -u username -p password
+              [-r realm] [-s servicename]
+              [-f socket path] [-R repeatnum]
+
+.SH DESCRIPTION
+This tool is for testing the saslauthd daemon. Do not use it unless you 
+know what you are doing. Read the source code if you need more information.
diff -urNad trunk~/utils/sasldbconverter2.8 trunk/utils/sasldbconverter2.8
--- trunk~/utils/sasldbconverter2.8	1970-01-01 02:00:00.000000000 +0200
+++ trunk/utils/sasldbconverter2.8	2006-07-27 16:20:07.000000000 +0300
@@ -0,0 +1,23 @@
+.\" sasldbconverter-2 - Convert sasldb file into the new format
+.\" Dima Barsky 25/10/2002
+.\"
+
+.\"
+.TH SASLDBCONVERTER2 8 "October 25, 2002" "CMU SASL"
+.SH NAME
+sasldbconverter2 \- Convert sasldb file into the new format
+.SH SYNOPSIS
+.B sasldbconverter2 [sasldb_file]
+.SH DESCRIPTION
+.I sasldbconverter2
+takes the sasldb file specified on the
+command line and convert it to a new sasldb file in the default
+location (usually /etc/sasldb2). It is STRONGLY RECOMMENDED that you
+backup sasldb before allowing this program to run
+.SH OPTIONS
+.TP
+.SH KNOWN BUGS
+This only works for sasldb files that use the gdbm library.
+.TP
+.SH SEE ALSO
+rfc2222 \- Simple Authentication and Security Layer (SASL)
diff -urNad trunk~/utils/sasltestsuite.8 trunk/utils/sasltestsuite.8
--- trunk~/utils/sasltestsuite.8	1970-01-01 02:00:00.000000000 +0200
+++ trunk/utils/sasltestsuite.8	2006-07-27 16:20:07.000000000 +0300
@@ -0,0 +1,22 @@
+.\" sasltestsuite - SASL2 test tool
+.\" Dima Barsky 23/11/2002
+.\"
+
+.\"
+.TH SASLTESTSUITE 8 "December 23, 2002" "CMU SASL"
+.SH NAME
+sasltestsuite \- SASL2 test tool
+.SH SYNOPSIS
+.B  sasltestsuite [-g name] [-s seed] [-r tests] -a -M
+    g -- gssapi service name to use (default: host)
+    r -- # of random tests to do (default: 25)
+    a -- do all corruption tests (and ignores random ones unless -r specified)
+    n -- skip the initial "do correctly" tests
+    h -- show this screen
+    s -- random seed to use
+    M -- detailed memory debugging ON
+
+.SH DESCRIPTION
+This tool is for testing the SASL2 installation. Do not use it
+unless you know what you are doing. Read the source code if you need 
+more information.
